### PR TITLE
Ci: docker compose for local test scenario

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description
Introduced a docker-compose for local test demonstration. Spins up following containers needed for edc 0.3.0
- one PURIS backend for customer and supplier each
- one PURIS frontend for customer and supplier each
- one control plane for customer and supplier each
- one data plane for customer and supplier each
- one postgresql
- one vault

It provides additional setup scripts and configuration for the given scenario.

Please note: fixed the secrets and then squashed. Tried multiple times as one path was seen as secret. Externalized all potential secrets into an .env

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by tom-rm-meyer-ISST** [comment]
Quality check still fails because we need to incorporate this issue: https://github.com/eclipse-tractusx/sig-infra/issues/230

**Comment by tom-rm-meyer-ISST** [comment]
Updated license headers.

**Comment by mhellmeier** [comment]
Regarding postgresql, remember to have a look at https://github.com/eclipse-tractusx/sig-infra/discussions/308

**Comment by tom-rm-meyer-ISST** [comment]
Regarding quality checks: The TRG for base images only applies to publishing images. As we use it locally for integration test, it's fine. I added the exclusion already to the metadata file. As soon as the reusable workflow is ready, it will pass. See [here](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md).

**Comment by tom-rm-meyer-ISST** [comment]
> Regarding quality checks: The TRG for base images only applies to publishing images. As we use it locally for integration test, it's fine. I added the exclusion already to the metadata file. As soon as the reusable workflow is ready, it will pass. See [here](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md).

Release guideline workflow has been updated. Everything works out now!

File: local/INSTALL.md
Comment (mhellmeier) [inline local/INSTALL.md:1]: Different points aren't necessary here.

File: local/daps/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '1'
```
Comment (mhellmeier) [inline local/daps/docker-compose.yaml:21]: Should be updated to version 3.

File: local/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
```
Comment (mhellmeier) [inline local/docker-compose.yaml:21]: Should be updated to version 3.

File: local/tractus-x-edc/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: "1"
```
Comment (mhellmeier) [inline local/tractus-x-edc/docker-compose.yaml:21]: Should be updated to version 3.

**Review by mhellmeier — CHANGES_REQUESTED** [review]
Different changes in the docker setup are necessary to get it working. I can fix it for you.

**Review by mhellmeier — APPROVED** [review]
LGTM, since it is only a local deployment for testing purposes and the KICS issues are addressed in #79

**Pull request merged**